src/admin-bro.ts

```diff
@@ -178,7 +178,7 @@ class AdminBro {
      */
     this.options = _.merge({}, defaults, options)
 
-    const defaultLogo = `${this.options.rootPath}/frontend/assets/logo-mini.svg`
+    const defaultLogo = path.join(this.options.rootPath, '/frontend/assets/logo-mini.svg')
```

Comment:
not sure but it can break on windows machines where path returns "\" instead of "/".

Do you have an option to test it on windows?

if it breakes you can use slash (which is in deps already) (`import slash from 'slash'`)